Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml

```diff
@@ -11,3 +11,17 @@
   - type: Sprite
     sprite: DeltaV/Objects/Misc/stamps.rsi
     state: stamp-psychologist
+
+- type: entity
+  name: administrative assistant rubber stamp
+  parent: RubberStampBase
+  id: RubberStampAdminAssistant
+  suffix: DO NOT MAP
```

Comment:
this should be on noSpawn: true if it's set to "do not map". Same with all the other stamps really I guess.